src/Client.js

```diff
@@ -0,0 +1,17 @@
+const httpClient = require('./http-client');
```

Comment:
Since this is a brand new library, can we use ES6 Modules' `import`, `export` syntax here?

Reply:
Yes, why not?